plugins/quickbars.md

```diff
@@ -74,46 +73,68 @@ tinymce.init({
     toolbar: false,
     menubar: false,
     inline: true,
-    quickbars_selection_toolbar: 'bold italic | quicklink h2 h3 blockquote'
+    quickbars_image_toolbar: false
 });
 ```
 
-#### Quick Image
+## Plugin-specific toolbar buttons
 
-The Quick Image (`quickimage`) control lets you quickly insert images from the local computer into the editor. These can then be automatically uploaded if you configure [image uploading]({{ site.baseurl }}/advanced/handle-async-image-uploads/). It can be used in both the Quick Insert toolbar and other toolbars.
+### Quick Link
 
-##### Example:
+The Quick Link (`quicklink`) toolbar button lets the user quickly insert/edit links inline. It is included in the Quick Selection context toolbar by default and can be used in other context toolbars.
+
+#### Example: Using quicklink in a custom context toolbar
 
 ```js
 tinymce.init({
     selector: "div.tinymce",
     plugins: [ 'quickbars' ],
-    toolbar: false,
-    menubar: false,
-    inline: true,
-    quickbars_insert_toolbar: 'quickimage quicktable'
+    setup: function(editor) {
+        editor.ui.registry.addContextToolbar('imageselection', {
+            predicate: function(node) {
+                return node.nodeName === 'P';
```

Comment:
should this be nodename === 'img' ?

Reply:
no, this is for quicklink not quickimage